### ISSUE TITLE
don't show d3mIndex plot in modelResults component

### ISSUE THREAD

**Issue by curtislisle** [d3m-purdue/modsquad2#6]
during a regression test, the target is compared against all other variables. We should delete the d3mIndex plot, though. as this would be confusing to a user.  It shows up if the source data has this column.

**Comment by hafen** [comment]
Good point and should be pretty easy to do. I'll look at this tomorrow as well.
